version/1/4/th/code_of_conduct.md

```diff
@@ -0,0 +1,50 @@
+# จรรยาบรรณฉบับของกติกามารยาท 
+
+## คำมั่นสัญญาของเรา
+
+เพื่อส่งเสริมสภาพแวดล้อมที่เปิดกว้างและเป็นที่ยอมรับ 
+เราในฐานะผู้ร่วมให้การสนับสนุนและผู้ดูแล สัญญาว่าในโครงการและชุมชนของเราจะเป็นประสบการณ์ ที่ปราศจากอคติสำหรับทุกคน โดยไม่คำนึงถึง อายุ ขนาดของร่างกาย ความพิการ ชาติพันธุ์, เอกลักษณ์ทางเพศและการแสดงออก, ระดับประสบการณ์,สัญชาติ เชื้อชาติ และ ศาสนา
+
+## มาตรฐานของเรา
+
+ตัวอย่างของพฤติกรรมที่ก่อให้เกิดสภาพแวดล้อมที่ดี
+* ใช้ภาษาที่แสดงถึงการทำงานเป็นทีม ที่กระตุ้นให้เกิดสิ่งแวดล้อมที่ดี
+* เคารพในมุมมองและประสบการณ์ที่ต่างกัน
+* ยอมรับอย่างสุภาพในเชิงวิจารณ์ที่สร้างสรรค์
+* มุ่งเน้นที่สิ่งที่ดีที่สุดสำหรับชุมชน
+* แสดงความเห็นอกเห็นใจต่อสมาชิกชุมชนคนอื่น ๆ
+
+ตัวอย่างของพฤติกรรมที่ไม่สามารถยอมรับได้โดยผู้เข้าอบรม
+* การใช้ภาษาหรือภาพ ที่ส่อถึงการคุกคามทางเพศ 
+*การแสดงความคิดเห็น เพื่อการโจมตีส่วนบุคคลหรือทางการเมือง โดยไม่คำนึงถึงเหตุและผล
+* การล่วงละเมิดในที่สาธารณะหรือส่วนตัว
+* เผยแพร่ข้อมูลส่วนตัวของผู้อื่นเช่น ที่อยู่ อีเมล์ เฟสบุ๊ค โดยไม่ได้รับอนุญาต
+*พฤติกรรมอื่น ๆ ที่อาจถือว่าไม่เหมาะสมในการทำงานอย่างมืออาชีพ
+
+## ความรับผิดชอบของเรา
+
+ผู้ดูแลโครงการมีหน้าที่ในการชี้แจงมาตรฐานของพฤติกรรมที่เหมาะสม และมีบทลงโทษที่ยุติธรรมสำหรับการไม่ปฏิบัติตามกฎข้อตกลงดังกล่าวที่กำหนดไว้
+
+ผู้ดูแลโครงการมีสิทธิ์และหน้าที่รับผิดชอบในการลบ แก้ไขหรือ ปฏิเสธความคิดเห็น, commits, wiki edits โค้ด, , ปัญหาต่างๆและผลงานอื่น ๆที่ไม่สอดคล้องกับหลักจรรยาบรรณนี้หรือ ไม่อนุญาตให้มีส่วนร่วมอย่างชั่วคราวหรือถาวรสำหรับพฤติกรรมอื่น ๆ ที่เห็นว่าไม่เหมาะสม,เชิงข่มขู่, ล่วงละเมิดหรือเป็นอันตราย
+
+## ขอบเขต 
+
+จรรยาบรรณนี้นำไปใช้ทั้งในขณะที่ทำงานและในขณะที่เผยแพร่สู่สาธารณะเมื่อบุคคลเป็นตัวแทนโครงการหรือชุมชนของตน
+ ตัวอย่างของการเป็นตัวแทนของโครงการหรือชุมชน ได้แก่ การใช้อีเมลโครงการอย่างเป็นทางการ ที่อยู่การโพสต์ผ่านทางการบัญชีสื่อสังคมออนไลน์ หรือทำหน้าที่เป็นผู้แต่งตั้งตัวแทนในงานออนไลน์หรือออฟไลน์
+ การเป็นตัวแทนของโครงการอาจเป็นได้เพิ่มเติม ซึ่งจะกำหนดและชี้แจงโดยผู้ดูแลโครงการ
+
+## การบังคับใช้
+ตัวอย่างของพฤติกรรมที่ไม่เหมาะสมล่วงละเมิดหรืออื่น ๆ ที่ยอมรับไม่ได้อาจได้รับการ
+รายงานโดยติดต่อทีมงานโครงการที่ [ใส่ที่อยู่อีเมล] 
+การร้องเรียนทั้งหมดจะได้รับการตรวจสอบและสอบสวน และจะมีผลตอบสนองอย่างเหมาะสมกับสภาพการณ์
+ ทีมงานโครงการจำเป็นต้องรักษาความลับเกี่ยวกับผู้ให้ข้อมูลของเหตุการณ์
```

Comment:
Unwanted space character at first index here.
All of comments here in this file also apply to other files in this PR.